bootstrap/src/main/kotlin/jp/co/soramitsu/notary/bootstrap/genesis/d3/D3TestGenesisFactory.kt

```diff
@@ -0,0 +1,118 @@
+package jp.co.soramitsu.notary.bootstrap.genesis.d3
+
+import com.google.protobuf.util.JsonFormat
+import jp.co.soramitsu.iroha.java.Transaction
+import jp.co.soramitsu.iroha.java.TransactionBuilder
+import jp.co.soramitsu.iroha.testcontainers.detail.GenesisBlockBuilder
+import jp.co.soramitsu.notary.bootstrap.dto.AccountPrototype
+import jp.co.soramitsu.notary.bootstrap.dto.IrohaAccountDto
+import jp.co.soramitsu.notary.bootstrap.dto.Peer
+import jp.co.soramitsu.notary.bootstrap.error.AccountException
+import jp.co.soramitsu.notary.bootstrap.genesis.*
+import java.util.*
+
+class D3TestGenesisFactory : GenesisInterface {
+    override fun getAccountsNeeded(): List<AccountPrototype> = D3TestContext.d3neededAccounts
+
+    override fun getProject(): String {
```

Comment:
Can we just write `override fun getProject() = "D3"` instead?

Reply:
fixed